src/sentry/static/sentry/app/views/eventsV2/table/tableView.tsx

```diff
@@ -104,19 +109,18 @@ class TableView extends React.Component<TableViewProps> {
 
   _renderGridHeaderCell = (column: TableColumn<keyof TableDataRow>): React.ReactNode => {
     const {eventView, location, tableData} = this.props;
+    const tableMeta = tableData && tableData.meta ? tableData.meta : undefined;
```

Comment:
I suppose this isn't equivalent if `meta` is falsey and not undefined

```suggestion
    const tableMeta = tableData?.meta;
```